app/javascript/inertia/admin_app_wrapper.tsx

```diff
@@ -33,30 +34,32 @@ export type GlobalProps = {
 
 const AdminAppWrapper = ({ children, global }: { children: React.ReactNode; global: GlobalProps }) => (
   <DesignContextProvider value={global.design_settings}>
-    <DomainSettingsProvider
-      value={{
-        scheme: global.domain_settings.scheme,
-        appDomain: global.domain_settings.app_domain,
-        rootDomain: global.domain_settings.root_domain,
-        shortDomain: global.domain_settings.short_domain,
-        discoverDomain: global.domain_settings.discover_domain,
-        thirdPartyAnalyticsDomain: global.domain_settings.third_party_analytics_domain,
-        apiDomain: global.domain_settings.api_domain,
-      }}
-    >
-      <UserAgentProvider
+    <TooltipProvider delayDuration={0} disableHoverableContent>
```

Comment:
This is needed by Radix. By default it has a delay on hover and some "grace area" around it which breaks tests, so turning those two off here.